src/parser.py

```diff
@@ -333,6 +366,25 @@ def _consume_or_raise(self, token_types, exception_description):
             return self._next_token()
         raise ParseError(token=self._peek(), message=exception_description)
 
+    def _function_call(self, function):
+        """
+        Gathers function and all arguments into a Call expression
+        """
+        arguments = []
+        if not self._is_same_type(TokenType.RIGHT_PAREN):
+            arguments.append(self._expression())
+            while self._is_one_of_types([TokenType.COMMA]):
+                if len(arguments) > 255:
+                    self.error_handler.report_error(ParseError(
```

Comment:
We probably want to raise an exception here. Handing / resynchronization happens at the top level.

Reply:
As discussed on the call will not save params/args > 255 and only add a single error